src/features/channels/components/channel-list/about/about.tsx

```diff
@@ -42,7 +63,24 @@ const About = ({ title, message, onClick }: Props) => {
       >
         <DialogTitle id="alert-dialog-title">{title}</DialogTitle>
         <DialogContent>
-          <DialogContentText id="alert-dialog-description">{message}</DialogContentText>
+          <DialogContentText id="alert-dialog-description">
+            <Grid className={styles.infoBox}>
+              <Divider />
+              {info &&
+                info.map((item) => {
```

Comment:
You can destructure the object like this and return directly the component
```suggestion
                info.map(({ name, version }) => 
                    <Grid>
                    ...
                     </Grid>
                    
```